tests/src/python/test_qgsrelation.py

```diff
@@ -196,6 +196,13 @@ def testValidRelationAfterChangingStyle(self):
         self.assertTrue(valid)
 
         # update style
+        # Note: the project is re-read because of a subtle bug with bidings involving
```

Comment:
```suggestion
        # Note: the project is re-read because of a subtle bug with bindings involving
```
....? looks like the meat of this note is missing !

Reply:
Yeah, it was an explanation of https://github.com/qgis/QGIS/pull/47671/files#r825704937 I'll re-add it.